tests/spec/tests/init.spec.js

```diff
@@ -0,0 +1,79 @@
+const http = require('http');
+const cmd = require('../helpers/cmd');
+const rimraf = require('rimraf');
+const { existsSync, readFileSync } = require('fs');
+const yaml = require('js-yaml');
+
+describe('hs init', () => {
+  const { cli, config } = global;
+
+  it('should print the correct output for personal access key init', async () => {
+    rimraf.sync('hubspot.config.yml');
+    expect(existsSync('hubspot.config.yml')).toBe(false);
```

Comment:
I don't have a great solution for this, but if either of these init tests fail, will that result in all the other tests also failing (since we're deleting the `hubspot.config.yaml` file)? Thinking about it a little more, maybe that's OK since it's such a core command?

Reply:
I moved the `rimraf` to the `beforeAll` method, but the rest of the tests will all still fail if the actual `hs init` command test fails. I think this is fine as that is the core of what we are testing here.